Apps/MAPL_GridCompSpecs_ACG.py

```diff
@@ -339,6 +359,19 @@ def emit_pointer_alloc(self):
             listout = EMPTY_LIST
         return listout
 
+    def emit_get_allocatable_variable(self):
+        args = [('itemName', self.spec_values[Option.SHORT_NAME])]
+        other_options = (Option[name] for name in 'STATE CONFIG INTERNAL_NAME'.split())
+        other_args = [(option.name, self.spec_values[option]) for option in other_options]
+        args.extend(other_args)
+        delimiter = MAPL_DataSpec.DELIMITER
+        procedure_call = make_procedure_call("StateFilterItem", delimiter=delimiter,
```

Comment:
This will not hold up the immediate PR,  but ...   I think the plan is to use the `MAPL_` prefix on all MAPL services.  So `StateFilterItem` needs a rename (in the `state/API.F90` file) to `MAPL_StateFilterItem`.   (And ACG needs to be updated in the same PR.)

Reply:
Noted.